lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +244,95 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Distribution:
+
+    """Wrapper class for pkg_resources.Requirement.
```

Comment:
It's an implementation detail. The docstring title should tell the reader class' purpose rather than which techniques you used. I'd go for `Python distribution package metadata wrapper.`